crates/astria-sequencer/src/app.rs

```diff
@@ -372,6 +380,10 @@ impl App {
 
             // don't include tx if it would make the cometBFT block too large
             if !block_size_constraints.cometbft_has_space(tx.len()) {
+                metrics::counter!(
+                    metrics_init::PREPARE_PROPOSAL_EXCLUDED_TRANSACTIONS_COMETBFT_SPACE
+                )
+                .increment(1);
```

Comment:
is this reset every block or is it a total over time?

Reply:
the counters accumulate over time in general

Reply:
I left them as counters because I didn't want to add variables to the function to track them per block production, the function is long already.